.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.sh text eol=lf
```

Comment:
Everything's run in a linux container, which gets really confused by all the `\r`'s if git auto-converts line endings to Windows, so we'll [deal with line endings](https://help.github.com/articles/dealing-with-line-endings/).